modules/ROOT/pages/design-assigning-permissions.adoc

```diff
@@ -0,0 +1,59 @@
+
+Depending on your organization's permission structure, you can assign permissions using the xref:access-management::teams.adoc[Teams] feature, the xref:access-management::users.adoc#grant-user-permissions[Roles] feature (deprecated), or by assigning an xref:access-management::users.adoc#grant-user-permissions[individual user permission].
+
+== Design Center Permissions
+
+=== Business Group Permissions
+
+Design Center Viewer:: Enables users to view all Design Center projects within a business group. Users with this permission cannot create new projects, edit or rename existing projects, or share projects with another user. Users with this permission can test a project with the Mocking Service. Assign this permission to those who consume your project in a specific business group.
+Design Center Creator:: Enables users to create projects in Design Center from the navigation panel, and view all projects created or shared with the user. Use this permission to invite users to create, edit and maintain your projects.
+Design Center Developer:: Enables users to view, create, and manage all projects within a business group. Use this permission to set up administrators for all the projects within a specific business group.
+
+=== Project Level Permissions
+
+Project Viewer:: Enables users to view a Design Center project within a business group. Users with this permission cannot create a new project, edit or rename the existing project, or share the project with another user. Users with this permission can test a project with the Mocking Service. Assign this permission to those who consume your project in a specific business group.
+Project Editor:: Enables users to edit a Design Center project within a business group. Use this permission to invite users to create, edit and maintain your projects.
+Project Administrator:: Enables users to manage and share a Design Center project within a business group. Use this permission to set up administrators for all the projects within a specific business group.
+
+
+NOTE: If you are granted organization and project permissions, the highest permission level will be the one that applies to that specific project.
+
+== Project Views
+
+From the Design Center home page, you can select between three types of views for your project list:
+
+*API Projects*: Lists all the projects you have access to.
+*Shared with me*: Lists only the projects shared with you by other users.
+*Mule flows*: Lists the Mule applications created by you. This view is only available if you have the “Design Center Developer” permission.
+
+== Sharing Projects from Design Center
+
+NOTE: Mule Applications can't be shared.
+
+You can share your projects and add the corresponding permissions directly from Design Center. To do so:
+
+* Go to the Design Center home page and select a project from the list.
+* On the *API Projects* header, click *Share Project*, or, under project details click *Change access*.
```

Comment:
Not sure what you mean by "header". On qax, when I'm in a project, I see a Share button at the top. I don't see Change access. Maybe a screenshot would help?